apps/figma-broker/README.md

```diff
@@ -5,15 +5,34 @@
 
 Extracts design decisions into [design tokens] from Figma using the [Figma Web API]
 
-# Notes 
+## Usage
+
+### Setup
+
+1. Add an .env file in the root.
+2. Go to Figma and generate a new personal access token (Found under your account settings).
+3. Set `FIGMA_TOKEN` env variable to the new personal access token.
+
+### Run
+
+1. Run either `tokens` or `assets` script with a `fileId` and/or flag for forcing to fetch new data from Figma.
+
+#### Example
+
+```sh
+pnpm run tokens 0TbIXrrObWj80Cf7KucKYFL0 true  # Fetches new data from Figma
+pnpm run tokens 0TbIXrrObWj80Cf7KucKYFL0 # Reads local figma json file
```

Comment:
This needs some clarification, it would be helpful to know how to find the fileId, what the local figma json file is, how it works etc. Also, I get an unhandled rejection error if I don’t have a local json file.

Reply:
I'll look into it 🤔

Reply:
What kind of error did you get? Worked fine here, running locally... 🤔 @vnys

Reply:
When I run the same command now there are no errors 🤷🏻‍♂️ Not sure why. But I got the error message mentioned earlier today if I didn’t have a local figma json file.

Reply:
You can delete the `raw` folder and try again? @vnys

Reply:
I did, but can’t reproduce the error – so just ignore it for now